model-optimizer/extensions/middle/ONNXResize10ToInterpolateV4.py

```diff
@@ -0,0 +1,138 @@
+    scales_ss = create_op_with_const_inputs(graph, StridedSlice,
+                                            {1: int64_array([begin_dim]),
+                                             2: int64_array([end_dim]),
+                                             3: int64_array([1])},
+                                            {'name': resize_name + '/scales_ss_',
+                                             'begin_mask': int64_array([1]),
+                                             'end_mask': int64_array([1]),
+                                             'new_axis_mask': int64_array([0]),
+                                             'shrink_axis_mask': int64_array([0]),
+                                             'ellipsis_mask': int64_array([0])})
+    axes_node = Const(graph,
+                      {'name': resize_name + '/axis_',
+                       'value': int64_array(np.arange(begin_dim, end_dim))}).create_node()
+
+    interpolate_node = Interpolate(graph, {'version': 'opset4',
+                                           'mode': 'linear_onnx' if resize.mode == 'linear' else 'nearest',
+                                           'coordinate_transformation_mode': 'asymmetric',
+                                           'cube_coeff': -0.75,
+                                           'nearest_mode': 'simple',
+                                           'pads_begin': int64_array([0]),
+                                           'pads_end': int64_array([0]),
+                                           'antialias': 0,
+                                           'shape_calculation_mode': 'scales',
+                                           'in_ports_count': 4}).create_node()
+
+    axes_node.out_port(0).connect(interpolate_node.in_port(3))
+    shape_of = Shape(graph, {'name': resize_name + '/ShapeOf_'}).create_node()
+
+    add_node = create_op_with_const_inputs(graph, Add,
+                                           {1: float_array([1.0e-5])},
+                                           {'name': resize_name + '/Add_'})
+
+    input_data_type = data_type_str_to_np(graph.graph['cmd_params'].data_type)
+
+    cast_shape_to_float = Cast(graph, {'dst_type': input_data_type}).create_node()
+    mul_node = Mul(graph, {'name': resize_name + '/Mul_'}).create_node()
+    shape_of.out_port(0).connect(cast_shape_to_float.in_port(0))
+    cast_shape_to_float.out_port(0).connect(mul_node.in_port(0))
+    cast_add_result_to_int = Cast(graph, {'dst_type': np.int64}).create_node()
+    floor_node = Floor(graph, {'name': resize_name + '/Floor_'}).create_node()
+    mul_node.out_port(0).connect(add_node.in_port(0))
+    add_node.out_port(0).connect(floor_node.in_port(0))
+    floor_node.out_port(0).connect(cast_add_result_to_int.in_port(0))
+    cast_add_result_to_int.out_port(0).connect(sizes_ss.in_port(0))
+    sizes_ss.out_port(0).connect(interpolate_node.in_port(1))
+    scales_ss.out_port(0).connect(interpolate_node.in_port(2))
```

Comment:
I may be wrong but here we perform the following `input_shape * scale + eps`. But we recently agreed that it should be `input_shape * (scale + eps)`. Please, change the MO code accordingly

Reply:
Done.